### PR TITLE
Fix/method complexity render input

### PR DESCRIPTION
**Pull request by JulianoGTZ**
It was the first time I had contact with the code climate, and I did not perform the analysis locally ... I was guided only by concepts of cognitive overload of the code.

I separated some responsibilities and rename some functions ... always executing all the tests to know if it was in the correct way.

Please note any snippet of code that is very dissonant from the one practiced in the repository. I want to contribute and keep the same readability.

### CODE REVIEW

File: src/utils/dom/renderers/renderInput.js [hunk 1]
```diff
@@ -3,36 +3,45 @@ import { warn, error, isPromise } from '../../utils.js'
...
+  
+  const input = renderInputType[params.input](params)
+  dom.show(input)
+}
 
-  if (rerender) {
-    const input = renderInputType[params.input](params)
-    dom.show(input)
-  }
+const hideContainer = (container) => {
```
Comment (limonte) [inline src/utils/dom/renderers/renderInput.js]: unnecessary function

File: src/utils/dom/renderers/renderInput.js [hunk 2]
```diff
@@ -3,36 +3,45 @@ import { warn, error, isPromise } from '../../utils.js'
...
 export const renderInput = (instance, params) => {
+  const content = dom.getContent()
   const innerParams = privateProps.innerParams.get(instance)
   const rerender = !innerParams || params.input !== innerParams.input
-  const content = dom.getContent()
-  const inputTypes = ['input', 'file', 'range', 'select', 'radio', 'checkbox', 'textarea']
-  for (let i = 0; i < inputTypes.length; i++) {
-    const inputClass = swalClasses[inputTypes[i]]
-    const inputContainer = dom.getChildByClass(content, inputClass)
+  const hasInputOnParams = params.input != undefined
```
Comment (limonte) [inline src/utils/dom/renderers/renderInput.js]: unnecessary variable, checking `!params.input` will work the same job

File: src/utils/dom/renderers/renderInput.js (hunk 1 above)
Comment (JulianoGTZ) [reply src/utils/dom/renderers/renderInput.js]: removed.

File: src/utils/dom/renderers/renderInput.js (hunk 2 above)
Comment (JulianoGTZ) [reply src/utils/dom/renderers/renderInput.js]: nice. Removed.

**Pull request merged**